lib/resources/r4/provenance.yaml

```diff
@@ -0,0 +1,198 @@
+---
+name: Provenance
+field_name_base_url: https://hl7.org/fhir/R4/provenance-definitions.html#Provenance
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "Provenance"
+    }
+  note: <ul><li>resourceType must be Provenance.</li></ul>
+  url: https://hl7.org/fhir/R4/json.html#resources
+
+- name: target
+  required: 'Yes'
+  cardinality: 1..*
+  type: Reference
+  description: The Reference(s) that were generated or updated by the activity described in this resource.
+  action: create
+  example: |
+    {
+      "target": [
+        {
+          "reference": "DocumentReference/881049"
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Only one target may be provided.</li>
+      <li> Currently we only support AllergyIntolerance, Condition, DocumentReference, Encounter, Immunization,
```

Comment:
The ..... resources are supported.

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/371/commits/e87e0be1558e277ac583722d167d60828e55216a#diff-c841f56a5b6c5fe0183663c5db12e9aee76806703b60dd935a89ed701a1f37f8R35).